content_tmp/ansible/roles/requirements.yml

```diff
@@ -0,0 +1,2 @@
+---
+- src: xlab_si.nuage_remove_entity
```

Comment:
@Fryguy @agrare so, we should prepare the support for this. In which repo/step we should install these? I would expect that bin/update and setup should also update the roles.

@miha-plesko we should probably have version of each role listed explicitely, so we can control it in backports.